sdk/python/cmd/pulumi-language-python/language_test.go

```diff
@@ -240,6 +240,10 @@ func TestLanguage(t *testing.T) {
 		//nolint:paralleltest // These aren't yet safe to run in parallel
 		for _, tt := range tests.Tests {
 			tt := tt
+			if tt != "l2-parameterized-resource" {
+				continue
+			}
```

Comment:
?

Reply:
Leftover code from testing just `l2-parameterized-resource`, removed it ✅